Task 7/script.js

```diff
@@ -0,0 +1,16 @@
+/* ------------------------------ TASK 7 -----------------------------------
+Turimas "audi" objektas.
+
+Parašykite funkciją "showObjectKeys", kuri kaip argumentą priims objektą 
+ir grąžins visus jo "key" masyve.
+-------------------------------------------------------------------------- */
+const audi = {
+  make: "audi",
+  model: "A6",
+  year: 2005,
+  color: "white"
+};
+
+const showObjectKeys = Object.keys(audi);
```

Comment:
Atidžiau skaitykite sąlygas, nes buvo reikėjo sukurti funkciją su konkrečiu pavadinimu.
Taip parandate nemenką dalį galutinio balo, kai užduotis buvo be galo paprasta.